pkg/docker/util.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/openshift/source-to-image/pkg/util/user"
 )
 
-// DockerImageReference points to a Docker image.
-type DockerImageReference struct {
+// ImageReference points to a Docker image.
+type ImageReference struct {
```

Comment:
that feels like an overly obsessive lint suggestion, i'd rather not see us jump through hoops for lint over things like that.  among other things, there's no guarantee the package even gets imported as "docker", (especially with a package name like docker that's likely to be reused by other packages)